test/test_nn.py

```diff
@@ -1839,6 +1839,23 @@ def check():
         parameters.clear()
         check()
 
+        parameters['p17'] = Parameter(torch.randn(10, 10))
+        parameter_dict['p17'] = parameters['p17']
+        self.assertIs(parameters['p17'], parameter_dict.get('p17'))
+        temp_param = Parameter(torch.randn(10, 10))
+        self.assertIs(parameters['p17'], parameter_dict.get('p17', temp_param))
+        self.assertIs(None, parameter_dict.get('p18'))
+        self.assertIs(temp_param, parameter_dict.get('p18', temp_param))
+        check()
+
+        parameter_dict.fromkeys(['p19', 'p20'])
+        self.assertIs({'p19': None, 'p20': None}, parameter_dict)
+        check()
+
+        parameter_dict.fromkeys(['p19', 'p20'], temp_param)
+        self.assertIs({'p19': temp_param, 'p20': temp_param}, parameter_dict)
```

Comment:
this should be assertEquals. This fails because assertIs checks that they are the exact same object. Meanwhile assertEquals checks if they have the same values.